UPGRADING

```diff
@@ -51,6 +51,9 @@ PHP 8.4 UPGRADE NOTES
 - GMP:
   . The GMP class is now final and cannot be extended anymore.
     RFC: https://wiki.php.net/rfc/gmp-final
+  . Casting a GMP object to bool changed so that 0 becomes false and everything else
+    becomes true.
+    RFC: https://github.com/php/php-src/pull/15151
```

Comment:
The RFC part should link to the RFC on the Wiki not the GitHub PR

Reply:
Oops, I made a mistake. Thank you